node/src/main/scala/coop/rchain/node/EventsInfo.scala

```diff
@@ -39,16 +50,25 @@ object EventsInfo {
       )
     }
 
-    val dsl = org.http4s.dsl.Http4sDsl[F]
-    import dsl._
-    HttpRoutes.of[F] {
-      case GET -> Root =>
-        val data =
-          EventConsumer[F].consume
-            .map(rca => { transformRChainEvent(rca) })
-            .map(j => Text(j.noSpaces))
-        val noop: Pipe[F, WebSocketFrame, Unit] = _.evalMap(_ => Sync[F].unit)
-        WebSocketBuilder[F].build(data, noop)
-    }
+    for {
+      topic <- Topic[F, WebSocketFrame](startedEvent)
```

Comment:
why is there a need for a topic? what is it?

Reply:
I'll add a better description